mod/groups/views/default/groups/edit/tools.php

```diff
@@ -8,20 +8,27 @@
  * @package ElggGroups
  */
 
+$input = '';
 $tools = elgg_get_config("group_tool_options");
 if ($tools) {
 	usort($tools, create_function('$a, $b', 'return strcmp($a->label, $b->label);'));
-	
+
 	foreach ($tools as $group_option) {
 		$group_option_toggle_name = $group_option->name . "_enable";
 		$value = elgg_extract($group_option_toggle_name, $vars);
-		
-		echo elgg_format_element('div', null, elgg_view('input/checkbox', array(
+
+		$input .= elgg_format_element('div', [], elgg_view('input/checkbox', [
 			'name' => $group_option_toggle_name,
 			'value' => 'yes',
 			'default' => 'no',
 			'checked' => ($value === 'yes') ? true : false,
 			'label' => $group_option->label
-		)));
+		]));
 	}
+}
+
+if ($input) {
+	echo elgg_view('elements/forms/field', [

```

Comment:
We can create `input/groups/tools` view that would take a list of tools in and spit out checkboxes